exporters/otlp/lib/opentelemetry/exporters/otlp/version.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Exporters
+    module OTLP
+      ## Current OpenTelemetry OTLP exporter version
```

Comment:
idk if you want to indicate that this is 0.5.0 version of the proto as well, or is that implied by "exporter"

Reply:
It's really just the gem version. In general, we're currently bumping the versions of all the gems in this repo in lockstep, as we release new Alpha versions of the SDK and API gems.

I'm not sure how to convey the version of the proto that we support. Perhaps just in the README.md?

Reply:
hrm...it may be a non issue at the moment.  if 0.5.0 is guarunteeing that subsequent versions of the proto can still be parsed  and no one actually used otlp maybe it's ok.

Reply:
The proto has been declared "stable" for tracing, so in theory, changes should be backwards compatible from now on. I think we should document the supported version of the proto in the README and ensure we bump the major version when the supported version of the proto changes in an incompatible way (most likely for metrics at this point), and document the change in the release notes.

Reply:
I updated the README to clarify the supported version of the protocol.